### PR TITLE
Reduce usage of whitelist

### PR DESCRIPTION
**Pull request by markstory**
There are better words to use than whitelist in these contexts.

### CODE REVIEW

File: en/orm/entities.rst [hunk 1]
```diff
@@ -99,7 +99,7 @@ When using ``set()`` you can update multiple fields at once using an array::
 
 .. warning::
 
-    When updating entities with request data you should whitelist which fields
+    When updating entities with request data you should indicate which fields
```
Comment (othercorey) [inline en/orm/entities.rst:102]:
```suggestion
    When updating entities with request data you should configure which fields
```

Can we link to a section that explains how to do this?

File: en/orm/saving-data.rst [hunk 2]
```diff
@@ -610,7 +610,7 @@ data needs to be restructured so it can be put into the right fields.
 
 The ``Model.beforeMarshal`` event is triggered just at the start of the
 validation process, one of the reasons is that ``beforeMarshal`` is allowed to
-change the validation rules and the saving options, such as the field whitelist.
+change the validation rules and the saving options, such as the field list.
```
Comment (othercorey) [inline en/orm/saving-data.rst:613]: This run-on sentence needs to be broke up but github won't let me make a multi-line suggestions here.

File: en/tutorials-and-examples/bookmarks/part-two.rst [hunk 3]
```diff
@@ -105,7 +105,7 @@ well. Again, in the ``UsersController``, add the following code::
         return $this->redirect($this->Auth->logout());
     }
 
-This code whitelists the ``logout`` action as a public action, and implements
+This code flags the ``logout`` action as a public action, and implements
```
Comment (othercorey) [inline en/tutorials-and-examples/bookmarks/part-two.rst:108]:
```suggestion
This code configures the ``logout`` action as a public action and implements
```

File: en/orm/saving-data.rst (hunk 2 above)
Comment (markstory) [reply en/orm/saving-data.rst:613]: Can do that separately.

**Pull request merged**